tests/test_preprints.py

```diff
@@ -48,6 +48,16 @@ def test_create_preprint_from_landing(self, session, driver, landing_page, proje
         submit_page.upload_select_file.click()
         submit_page.upload_file_save_continue.click()
 
+        # Author assertions
+        submit_page.public_available_button.click()
+        submit_page.public_data_input.click()
+        submit_page.public_data_input.send_keys_deliberately('https://osf.io/')
+        submit_page.public_data_save.click()
```

Comment:
I don't think you need to click this button to save. Clicking this button adds another link field beneath the current one.